dfttopif/parsers/vasp.py

```diff
@@ -17,33 +17,52 @@ class VaspParser(DFTParser):
     def __init__(self, directory):
         super(VaspParser, self).__init__(directory)
 
-        if not 'OUTCAR' in os.listdir(self._directory):
-            raise InvalidIngesterException('OUTCAR not found')
+        # Get the list of files in this directory
+        files = [f for f in os.listdir(directory) if os.path.isfile(os.path.join(directory, f))]
+
+        # Find the outcar file
+        def _find_file(name):
+            """Find a filename that contains a certain string"""
+            for f in files:
+                if f.upper().startswith(name):
```

Comment:
This only works if `name` is all caps. How about `f.upper().startswith(name.upper())`

Reply:
👍 I'll make the change